k8s/helm-chart/dynatrace-gcp-function/templates/active-gate-pod.yaml

```diff
@@ -0,0 +1,108 @@
+# Copyright 2021 Dynatrace LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+{{- if or (eq .Values.deploymentType "logs") (eq .Values.deploymentType "all") }}
+  {{- if eq .Values.activeGate.useExisting "false" }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: dynatrace-activegate-gcpmon-clusterip
+  namespace: dynatrace
```

Comment:
also putting this is in each template is not good practice... because during installation you also setting namespace so some parts can be install in different namespace

Reply:
Namespace for Dynatrace resources as of now must not be different than dynatrace. (all resources including Operator, Agent are there). Having this in mind, dynamic namespace will rather obfuscate the deployment code :)

Reply:
ok so we should remove it from deployment command because we have hardcoded namespace in each template

Reply:
which deployment command do U refer to?

Reply:
https://github.com/dynatrace-oss/dynatrace-gcp-function/pull/118/files#diff-8c092cd24fc76c45c928dfac92e5f6593cf7854a3eb844c1b8dee57312eecfecR316

Reply:
ok forgot about helm :) actually I've double checked what we do in other helm charts currently and yes namespace is dynamic there https://github.com/Dynatrace/helm-charts/blob/master/dynatrace-operator/templates/Kubernetes/operator/deployment-operator.yaml . So actually I agree @karolkieglerski  with Your comment to make it variable here as well.